prow/e2e-cluster_wide-auth.sh

```diff
@@ -31,4 +31,4 @@ set -x
 ROOT=$(cd $(dirname $0)/..; pwd)
 
 echo 'Running cluster-wide e2e rbac, auth Tests'
-${ROOT}/prow/e2e-suite.sh --test_vm --auth_enable --cluster_wide "$@"
+${ROOT}/prow/e2e-suite.sh --test_vm --auth_enable --cluster_wide --use_galley_config_validator "$@"
```

Comment:
Is this a required suite ? If yes - please use a different one (until 1.0).

Reply:
Server-side validation is targeted beta for 1.0 (see https://github.com/istio/istio/issues/1894).